docs/site/content/en/docs/Tasks/import-export/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Import and Export Tests and Schemas
+date: 2023-11-30
+description: How to import and export Tests and Schemas in Horreum
+categories: [Tutorial]
+weight: 3
+---
+
+> **Prerequisites**:
+
+> 1. Horreum is running
+> 2. To export you have previously defined a `Schema` for the JSON data you wish to analyze, please see [Define a Schema](/docs/tasks/define-schema-and-views/)
+> 3. To export you have previously defined a Test, please see [Create new Test](/docs/tasks/create-new-test/)
+
+## Background
+
+To simplify copying Tests and Schemas between Horreum instances Horreum provides a simple API to export and import new Tests and Schemas. Horreum also support updating exising Schemas and Tests by importing Tests or Schemas with existing Id's.
+
+## TestExport
+
+The export object for Tests is called [TestExport](https://github.com/Hyperfoil/Horreum/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/TestExport.java) and contains a lot of other fields in addition to what's defined in [Test](https://github.com/Hyperfoil/Horreum/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/Test.java). This includes, variables, experiments, actions, subscriptions, datastore and missingDataRules. This is to simplify the import/export experience and make sure that all the data related to a Test has a single entrypoint with regards to import and export. Note that secrets defined on [Action](https://github.com/Hyperfoil/Horreum/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/Action.java) are not portable between Horreum instances and there might be security concerns so they are omitted. The apiKey and password attributs defined on the config field in [Datastore](https://github.com/Hyperfoil/Horreum/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/Datastore.java) are also omitted and will have to be manually added in a separate step.
+
+## TestSchema
+
+The export object for Schemas is called [SchemaExport](https://github.com/Hyperfoil/Horreum/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/SchemaExport.java) and contains other fields in addition to what's defined in [Schema](https://github.com/Hyperfoil/Horreum/horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/data/Schema.java). This includes, labels, extractors and transformers. This is to simplify the import/export experience and make sure that all the data related to a Schema has a single entrypoint with regards to import and export.
```

Comment:
For User docs we should focus on what the user sees, not implementation detail. We can describe the JSON structure (or better still link to the OpenAPI spec).

If we are talking about concepts `Test` / `Actions` etc, that should link to those descriptions in the User guide (i.e. https://horreum.hyperfoil.io/docs/concepts/core-concepts/), not the source code